generators/server/templates/src/main/java/package/web/rest/dto/_ManagedUserDTO.java

```diff
@@ -2,13 +2,20 @@
 <% if (databaseType == 'mongodb' || databaseType == 'sql') { %>
 import java.time.ZonedDateTime;<% } %>
 
+import java.util.Set;
+
 import <%=packageName%>.domain.User;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
 
 /**
  * A DTO extending the UserDTO, which is meant to be used in the user management UI.
  */
 public class ManagedUserDTO extends UserDTO {
 
+    public static final int PASSWORD_MIN_LENGTH = 5;
```

Comment:
Very well seen, indeed! :+1: 

@mraible You modified this min value in the commit 9460a762c0cc90c557f621347b8849ed52c0005a, do you remember why did you had to make this change? Something with JWT perhaps?